template.txt

```diff
@@ -0,0 +1,21 @@
+mapbotuser = "root"
+mapbotpassword = "root"
+mapbothost = "localhost"
+mapbotdatabase = "mapbot"
+mapbotport = "3306"
+mapbotkey = "*Google_Cloud_API_key*"   # Will be provided by mentors
+
+# your_path_to_stanford-corenlp-x.x.x.jar
+stanford_path_to_jar = "./stanford-corenlp-full-2018-10-05/stanford-corenlp-3.9.2.jar"
+
+# your_path_to_stanford-corenlp-x.x.x-models.jar
+stanford_path_to_models_jar = "./stanford-corenlp-full-2018-10-05/stanford-corenlp-3.9.2-models.jar"
+
+# for eg. 'C:\\Program\ Files\\Java\\jdk1.8.0_201\\bin\\java.exe' or '/usr/local/openjdk-11/bin/java'
+javahome = '*your_path_to_jdk_bin_java.exe*'
+
+#DOCKER variables
```

Comment:
set DOCKER=Y for these variables to be picked up

Reply:
I did not understand..what do I have to do?

Reply:
@Varshney200 the `if` condition that we've used in the .env file should be present in the template too, since a template is just meant to be filled and modified as less as possible. It seems you forgot that line.

Reply:
The config.py  script has the if statement.. should it be added in the .env file too?

Reply:
I think what @vishakha-lall means to add is declare a `DOCKER` environment variable in the `.env` file itself, so that the conditional in `config.py` is handled correctly.

What I mentioned was a bit faulty. I misread the `template.txt` as `config.py`. You didn't forget a line. No worries.

Reply:
what should the `DOCKER `variable be initialised with?

Reply:
Certainly, I do mean that the variable must be initialised as something, and a modification in `config.py` to check for variable value Y, Y indicating a docker setup and N indicating regular setup. 
When DOCKER=Y here, `config.py` would take the docker values and when it's set to N here, it shouldn't.

Reply:
done